cmd/nfd-topology-updater/main.go

```diff
@@ -86,6 +113,14 @@ func parseArgs(flags *flag.FlagSet, osArgs ...string) (*topology.Args, *resource
 		os.Exit(2)
 	}
 
+	if len(resourcemonitorArgs.KubeletConfigURI) == 0 {
+		if len(nfdclient.NodeName()) == 0 {
+			fmt.Fprintf(flags.Output(), "environment variable NODE_NAME must be defined")
+			os.Exit(1)
```

Comment:
nit:
1. add `\n``
2. Maybe we could be more specific what we want that env for.

Probably not perfect but e.g. smth like


```suggestion
			fmt.Fprintf(flags.Output(), "unable to determine the default 'https://${NODE_NAME}:%d/configz endpoint', either define the NODE_NAME environment variable or specify endpoint with the -kubelet-config-uri flag\n", kubeletSecurePort)
			os.Exit(1)
```

WDYT?